Framework/PythonInterface/mantid/api/src/Exports/ITableWorkspace.cpp

```diff
@@ -677,6 +692,14 @@ void export_ITableWorkspace() {
            "number then it is interpreted as a row otherwise it "
            "is interpreted as a column name.")
 
+      .def("setCell", &setCellNoNotify,
```

Comment:
This will have changed the call signature on the Python side so that it requires the extra argument. We can use `BOOST_PYTHON_FUNCTION_OVERLOADS` to provide a default argument. Have a look at the [`WorkspaceFactory`](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/mantid/api/src/Exports/WorkspaceFactory.cpp#L60) export